src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback, useState } from "react";
+import classNames from "classnames";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronRight,
+  faChevronLeft,
+  faCommentDots,
+} from "@fortawesome/free-solid-svg-icons";
+
+import UserProfileModal from "components/organisms/UserProfileModal";
+import { VenueChat, PrivateChats } from "./components";
+
+import {
+  useChatsSidebarControls,
+  useChatsSidebarInfo,
+} from "hooks/useChatsSidebar";
+
+import { ChatTypes } from "types/chat";
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+
+import "./ChatSidebar.scss";
+
+export const ChatSidebar: React.FC = () => {
+  const [selectedProfile, setSetSelectedProfile] = useState<WithId<User>>();
+  const unselectProfile = useCallback(() => setSetSelectedProfile(undefined), [
+    setSetSelectedProfile,
+  ]);
+  const hasSelectedProfile = selectedProfile !== undefined;
+
+  const {
+    isChatSidebarVisible,
+    chatSettings,
+
+    openVenueChat,
+    openPrivateChats,
+    closeChat,
+    expandChat,
+  } = useChatsSidebarControls();
+
+  const { privateChatTabTitle, venueChatTabTitle } = useChatsSidebarInfo();
```

Comment:
Good question. I don't necessarily have answer for that. Keep in mind that it will have more logic inside. For example, deciding the correct name for `Venue chat`

Reply:
Fair enough.. I guess we'll see how I think/feel about it when I see it with all it's logic.